datadog_checks_dev/datadog_checks/dev/tooling/templates/check/{check_name}/README.md

```diff
@@ -0,0 +1,47 @@
+# Agent Check: {check_name_cap}
+## Overview
+
+This check monitors [{check_name_cap}][1].
+
+## Setup
+
+### Installation
+
+The {check_name_cap} check is included in the [Datadog Agent][2] package, so you don't
+need to install anything else on your server.
+
+### Configuration
+
+1. Edit the `{check_name}.d/conf.yaml` file, in the `conf.d/` folder at the root of your
+  Agent's configuration directory to start collecting your {check_name} performance data.
+  See the [sample {check_name}.d/conf.yaml][3] for all available configuration options.
+
+2. [Restart the Agent][4]
+
+### Validation
+
+[Run the Agent's `status` subcommand][5] and look for `{check_name}` under the Checks section.
+
+## Data Collected
+### Metrics
+
+The {check_name_cap} check does not include any metrics at this time.
```

Comment:
We don't need to say "at this time" here. The documentation represents _now_, not a hypothetical future.
> The {check_name_cap} check does not include any metrics.

Imho, we could even shorten this to:
> {check_name_cap} does not include any metrics.